node/node.go

```diff
@@ -1172,7 +1172,7 @@ func (n *Node) getAccountRootHashAndVal(address []byte, accBytes []byte, key []b
 		return nil, nil, fmt.Errorf("empty dataTrie rootHash")
 	}
 
-	retrievedVal, err := userAccount.RetrieveValueFromDataTrieTracker(key)
+	retrievedVal, err := userAccount.RetrieveValue(key)
```

Comment:
when system-testing this branch, please make sure you test the `/address/erd1.../esdt` endpoint. it should validate that the affected functionalities (save key value, retrieve value) work the same

Reply:
It works as expected.